src/odemis/gui/comp/canvas.py

```diff
@@ -983,7 +983,9 @@ def _draw_merged_images(self, ctx, interpolate_data=False):
                     #     1         0      1
                     # TODO: for now, this only works correctly if the background
                     # is black (otherwise, the background is also mixed in)
-                    if bm_last == BLEND_SCREEN:
+                    if 'merge_ratio' in md:
```

Comment:
There is another function that implement the same logic: `gui.util.img.images_to_export_data()`. This is used when calling "Export". Admittedly, it's not used on the FASTEM, so not a big deal, but please update that function as well, so that it keeps doing the same thing (and we don't get surprise later).

Reply:
Didn't know this function. Updated!